### ISSUE TITLE
problems with WireBox

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ohms-law#113]
(1) WireBox is passing one set of options to `Node.call`, another set to `mutate`.  This is an anti-pattern, and will potentially cause initialization problems with PhET-iO.

(2) The assumption documented on line 92, `this assumes that it is a child of a 'ul'`, should be backed up with an assertion.  Something like `assert && assert( options.tagName === 'ul', ... )`.

**Comment by zepumph** [comment]
Good comments, thanks.
